### ISSUE TITLE
Sentiment Analysis in Rasa

### ISSUE THREAD

**Issue by znat** [RasaHQ/rasa_nlu#425]
Would be really great to have that and I'm interested in contributing. For example, I like the way [recast.ai](https://recast.ai/) adds sentiment to any query.

This is to start the discussion: what should it be? 
Should it be a (1) pre-trained per language model with state of the art results like models trained on the Stanford sentiment tree bank, or a (2) trained in Rasa model? (a sentiment label would be added to each example fed to Rasa at training time).

(1) would probably complicate the stack a lot (recursive networks not well supported in DL frameworks) but there a not state of the art models that are is still good and easy to implement in Keras. But does it make sense to add DL to the Rasa stack?

(2) seems easier as I suppose decent results on small datasets can be obtained with algos supported by sklearn. Probably lower accuracy but it's also domain specific so it may be good enough. 

What are your thoughts?

**Comment by PHLF** [comment]
See https://github.com/RasaHQ/rasa_nlu/issues/230

**Comment by tmbo** [comment]
@znat Very good idea and I'd love to hear your input on this, and if you like, this would be an awesome contribution. Let's discuss on the existing issue though.
